Various fixes

MeuhV1 opened the pull request:
Fixed American UHV help text
Fixed wonder limit hoover help and city indicator
Burj Khalifa does not work on peaks

dguenms commented:
I'm ready to merge this, can you recompile the DLL with the latest updates? Or just remove the DLL from the PR and I'll compile during the merge.

Commit:
Fixed American UHV help text
Fixed wonder limit hoover help text and city indicator
Burj Khalifa does not work on peaks

MeuhV1 commented:
Updated.

I merged all changes in 1 commit, because that was more convenient for me.
